src/styles/blocks/header.scss

```diff
@@ -26,12 +28,18 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: transform $transition-time ease, color $transition-time ease;
+    cursor: pointer;
 
     &:not(:last-child) {
       margin-right: 20px;
     }
   }
 
+  &:hover {
+    color: $color-nav-item;
+  }
```

Comment:
I think it should be under __item element, because hover on header item still isn't working